.github/workflows/docker_image_push.yml

```diff
@@ -44,10 +44,14 @@ jobs:
       run: pytest ./tests/integration_tests/test_dataloader_ipfs.py
     - name: Code Coverage
       run: codecov
-    - name: Push image to DockerHub
-      env:
-        username: $DOCKER_USER
-        password: $DOCKERPASS
-      run: |
-        echo $DOCKERPASS | docker login --username $DOCKER_USER --password-stdin
-        ./scripts/push_image.sh
+    - name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+        username: ${{secrets.DOCKER_USER}}
+        password: ${{secrets.DOCKER_PASS}}
+    - name: Build and Push to Docker
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        push: true
+        tags: bittensor/bittensor:subtensor_tests
```

Comment:
why do we have this tag here? bittensor:subtensor_tests? shouldn't it just be bittensor:bittensor.

Reply:
@unconst do you mean `bittensor/bittensor:latest` ? So following the circle ci config, the push image script seems to target `bittensor/bittensor:subtensor_tests` 

`./scripts/build_protos.sh
docker build -t bittensor/bittensor:subtensor_tests -f Dockerfile .
docker push bittensor/bittensor:subtensor_tests`

Should I use `bittensor/bittensor:latest` instead?

Reply:
Yes I think you should yes. @shibshib  can you comment on this? Or was Parralax the prev owner here

Reply:
Definitely some remnant of parallaxs code, should use bittensor/bittensor:latest

Reply:
I deleted that subtensor thing

Reply:
made the necessary changes. @unconst @shibshib please review